src/project-2.js

```diff
@@ -44,62 +76,107 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+  if (num === 2) {
+    return true;
+  } else if (num === 0 || num === 1 || num % 2 === 0) {
+    return false;
+  }
+  for (let i = 3; i < num; i++) {
+    if (num % i === 0) {
+      return false;
+    }
+  }
+  return true;
 };
 
 const returnFirst = (arr) => {
   // return the first item from the array
+  return arr[0];
 };
 
 const returnLast = (arr) => {
   // return the last item of the array
+  return arr[arr.length - 1];
 };
 
 const getArrayLength = (arr) => {
   // return the length of the array
+  return arr.length;
 };
 
 const incrementByOne = (arr) => {
   // arr is an array of integers
   // increase each integer by one
   // return the array
+  for (let i = 0; i < arr.length; i++) {
+    arr[i] += 1;
+  }
+  return arr;
 };
 
 const addItemToArray = (arr, item) => {
   // add the item to the end of the array
   // return the array
+  arr.push(item);
+  return arr;
 };
 
 const addItemToFront = (arr, item) => {
   // add the item to the front of the array
   // return the array
   // hint: use the array method .unshift
+  arr.unshift(item);
+  return arr;
 };
 
 const wordsToSentence = (words) => {
   // words is an array of strings
   // return a string that is all of the words concatenated together
   // spaces need to be between each word
   // example: ['Hello', 'world!'] -> 'Hello world!'
+  let everyWord = words[0];
+  for (let i = 1; i < words.length; i++) {
+    everyWord += ` ${words[i]}`;
+  }
+  return everyWord;
 };
 
 const contains = (arr, item) => {
   // check to see if item is inside of arr
   // return true if it is, otherwise return false
+  return arr.includes(item);
 };
 
 const addNumbers = (numbers) => {
   // numbers is an array of integers.
   // add all of the integers and return the value
+  let sum = 0;
```

Comment:
This is also a great time to use `Array.reduce`.  It's fine that you didn't but now that you're becoming more advanced this is where you should use it